### ISSUE TITLE
Main table data and info

### ISSUE THREAD

**Issue by serejandmyself** [citizenweb3/validatorinfo#43]
## Tl;DR
- The main page contains the main data table of the website. It is the face of the idea and the project
- One of the goals is to remove tribalism amongst ecosystems, hence this is a multichain explorer for validators, showing validators across all possible POS networks and beyond (with the ability to filter by ecosystem)
- Second idea is to minimize the damage explorers, etc do to decentralization of stake, by minimizing unhealthy ranking by TVL (TVS in our table)
- Third is the introduction of information about validators in one place
- Fourthly, showing clear, important data that makes sense to all types of users
- The table offers filtering 
- By default, the sorting is random. Users may hardcode their own top in the dashboard stage

## The table

| Val. Name | Links | Battery | Tech. Score | Soc. Score | Gov. Score | User Score | TVS | Supported Assets |
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
| CW3 | Favicons | Indicator, based on badges | 0-100| 0-100 | 0-100 | 0-100 | Pie Chart breakdown | Logos | 

- Technical score, Social score, Governance score, User score (TS, SS, GS, US) are metrics we calculate on our side (explored in detail on the profile page - they are clickable links)
- Final result are calculation as a score based out of a 100 or 10 to make it easy
- Social links: Website, Twitter, GitHub. Shown as clickable favicons
- Supported assets: shown as logos of tokens or/and ecosystems in black and white
- Battery is based on badges collected for actions. At this stage it will be empty for all validators
- TVS is overall staked number with the validator across all networks 
- Also, TVS is shown as pie-chart, which shows the breakdown of the distribution a validator owns
- Example of pie chart: https://staking-explorer.com/staking/cosmoshub

## Calculated scores
1) Technical score
- To include in calculations: Total amount of networks, total amount of known nodes?, total voting power per network?, supported assets by validator, participation rate, uptime, amount of missed blocks per period, amount of status (active, etc), A reading of ISP providers and locations of nodes compares to others, Stakebar technical data

2) Social score
- To include in calculations: has twitter, has website, has logo, has description, twitter followers?, twitter lunarcrush or similar score?, has telegram, done interview with us, done other interviews, has info on profile page about their public goods, has revenue info on profile page, has keplr web3 reviews, has reviews on our site, other ratings: bro n bro and stakignrewards, public pgp keys?, twitter # reading?, rss hashtag readings?, other links?

3) Governance score
- To include in calculations: average commission per network compares to others?, amount of delegators, self stake ratios?, fans (account that delegate to them), amount of created proposals, amount of voted proposals, governance forum activity, social network administration

4) User score
- At start will be NA for all validators 
- To include in calculations: additions to favourites, amount staked on validator, minted nfts/poaps of validator, review score of validator, additions of users to validator page

**Comment by serejandmyself** [comment]
https://www.daometer.xyz/methodology
